code/modules/lighting/light_source.dm

```diff
@@ -169,6 +170,11 @@
 					round(lum_b * ., LIGHTING_ROUND_VALUE)
 				)
 
+			else
+				effect_r += 0
+				effect_b += 0
+				effect_g += 0
```

Comment:
Yes but that's a weird fucking syntax for adding a 0 as an element to a list really.